src/components/widgets/SelectWidget.js

```diff
@@ -32,6 +36,11 @@ function SelectWidget({
   onChange
 }) {
   const {enumOptions} = options;
+
+  const emptyOption = !multiple && (!schema.default || schema.default.length === 0)
```

Comment:
@kmills006 to confirm. This is meant to hide the empty option when a default is configured in the BSON schema or the default is configured as an empty string, correct? If that's the case, what if we show the empty option all the time unless it is a multiple select...?